game_options.cpp

```diff
@@ -159,5 +162,14 @@ void test_game_options()
   }
   #endif // FIX_ISSUE_353
 
+ #define FIX_ISSUE_383
+#ifdef FIX_ISSUE_383
+// A game_options has an environment_type member
+{
+  game_options go;

```

Comment:
The be very precise, this should be `const game_options go`, as reading/getting data should/does not modify the `game_options`